app/src/main/java/com/lastaoutdoor/lasta/ui/screen/social/components/FriendsList.kt

```diff
@@ -110,51 +111,42 @@ fun FriendsCard(friend: UserModel, navToFriend: () -> Unit) {
-                }
-            // display the user's sport preference
-            Row(modifier = Modifier.testTag("FriendPrefActivity")) {
-              val prefActivity = friend.prefActivity
-              val prefActivityLevel =
-                  when (prefActivity) {
-                    ActivityType.CLIMBING -> friend.levels.climbingLevel
-                    ActivityType.HIKING -> friend.levels.hikingLevel
-                    ActivityType.BIKING -> friend.levels.bikingLevel
+                  Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
+                        text = friend.language.resourcesToString(LocalContext.current),
+                        modifier = Modifier.padding(5.dp))
+                    Image(
+                        painter = friend.language.getIcon(),
+                        contentDescription = "language logo",
+                        modifier = Modifier.size(25.dp, 25.dp))
                   }
-              Text(
-                  modifier = Modifier.testTag("friendInfo"),
-                  text =
-                      when (prefActivity) {
-                        ActivityType.CLIMBING ->
-                            when (prefActivityLevel) {
-                              UserLevel.BEGINNER ->
-                                  LocalContext.current.getString(R.string.climbing_newcomer)
-                              UserLevel.INTERMEDIATE ->
-                                  LocalContext.current.getString(R.string.climbing_amateur)
-                              UserLevel.ADVANCED ->
-                                  LocalContext.current.getString(R.string.climbing_expert)
-                            }
-                        ActivityType.HIKING ->
-                            when (prefActivityLevel) {
-                              UserLevel.BEGINNER ->
-                                  LocalContext.current.getString(R.string.hiking_newcomer)
-                              UserLevel.INTERMEDIATE ->
-                                  LocalContext.current.getString(R.string.hiking_amateur)
-                              UserLevel.ADVANCED ->
-                                  LocalContext.current.getString(R.string.hiking_expert)
-                            }
-                        ActivityType.BIKING ->
-                            when (prefActivityLevel) {
-                              UserLevel.BEGINNER ->
-                                  LocalContext.current.getString(R.string.biking_newcomer)
-                              UserLevel.INTERMEDIATE ->
-                                  LocalContext.current.getString(R.string.biking_amateur)
-                              UserLevel.ADVANCED ->
-                                  LocalContext.current.getString(R.string.biking_expert)
-                            }
-                      })
-            }
+                }
           }
         }
+        Row(

```

Comment:
There's no padding added to this row, which makes content without margin and it looks overwhelming! Try adding a padding of 8.dp to this row as the one above it so it looks good & homogenous :)